### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by GaryFNC**
### What does this PR do?

Update CHANGELOG.md for release v41.16.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,22 @@
-##41.16.0 - January 8, 2018
+## 41.16.0 - January 8, 2018
 ### Added
-#### salesforce-vscode-apex
-* Update apex-jorje-lsp.jar to enable go to definition for type usage sites ([PR #247] (https://github.com/forcedotcom/salesforcedx-vscode/pull/247))
+#### All
+* Upgrade to TypeScript 2.6.2 ([PR #242] (https://github.com/forcedotcom/salesforcedx-vscode/pull/242))
```
Comment (ruthemmanuelle) [inline packages/salesforcedx-vscode/CHANGELOG.md]: The space between the square bracket and the parenthesis prevents the link from working. The spaces should be removed in all of these change log entries.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+#### salesforcedx-vscode-apex
+* Enable Go To Definition for usages of classes and interfaces in class and interface declarations ([PR #247] (https://github.com/forcedotcom/salesforcedx-vscode/pull/247))
+#### scripts
```
Comment (ruthemmanuelle) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Do users care about this item?

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,22 @@
...
+* Add script to enable testing with external dependencies ([PR #244] (https://github.com/forcedotcom/salesforcedx-vscode/pull/244))
+#### salesforcedx-apex-debugger
+* Add visual indication in the call stack telling the user what exception the user is currently paused on ([PR #240] (https://github.com/forcedotcom/salesforcedx-vscode/pull/240)
+#### salesforcedx-vscode-lwc
+* Add LWC Extension ([PR #241] (https://github.com/forcedotcom/salesforcedx-vscode/pull/241))
+#### salesforcedx-vscode
+* Add dreamforce video links to README ([PR #239] (https://github.com/forcedotcom/salesforcedx-vscode/pull/239))
```
Comment (ruthemmanuelle) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Dreamforce should be capitalized.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,22 @@
...
+## 41.16.0 - January 8, 2018
 ### Added
-#### salesforce-vscode-apex
-* Update apex-jorje-lsp.jar to enable go to definition for type usage sites ([PR #247] (https://github.com/forcedotcom/salesforcedx-vscode/pull/247))
+#### All
+* Upgrade to TypeScript 2.6.2 ([PR #242] (https://github.com/forcedotcom/salesforcedx-vscode/pull/242))
+#### salesforcedx-vscode-apex
+* Enable Go To Definition for usages of classes and interfaces in class and interface declarations ([PR #247] (https://github.com/forcedotcom/salesforcedx-vscode/pull/247))
+#### scripts
+* Add script to enable testing with external dependencies ([PR #244] (https://github.com/forcedotcom/salesforcedx-vscode/pull/244))
```
Comment (ruthemmanuelle) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Do users care about this item?

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 5]
```diff
@@ -1,7 +1,22 @@
...
+#### salesforcedx-apex-debugger
+* Add visual indication in the call stack telling the user what exception the user is currently paused on ([PR #240] (https://github.com/forcedotcom/salesforcedx-vscode/pull/240)
```
Comment (ruthemmanuelle) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Should "the user is currently paused on" be "the debugger is currently paused on"?

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 1 above)
Comment (vazexqi) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Remove this. It's not necessary and has not customer facing impact.

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 4 above)
Comment (vazexqi) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Remove this. It's not necessary and has not customer facing impact.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 6]
```diff
@@ -1,7 +1,22 @@
...
+#### salesforcedx-vscode-lwc
+* Add LWC Extension ([PR #241] (https://github.com/forcedotcom/salesforcedx-vscode/pull/241))
```
Comment (vazexqi) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Remove this. We will have a proper post about this later.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 7]
```diff
@@ -1,7 +1,22 @@
...
+#### salesforcedx-vscode
+* Add dreamforce video links to README ([PR #239] (https://github.com/forcedotcom/salesforcedx-vscode/pull/239))
+### Fixed
+#### All
+* Update dependency on vscode to 1.1.10 to enable testing against VS Code Insiders build on Linux ([PR #231] (https://github.com/forcedotcom/salesforcedx-vscode/pull/231))
```
Comment (vazexqi) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Remove this. It's not necessary and has not customer facing impact.

Comment (GaryFNC) [reply packages/salesforcedx-vscode/CHANGELOG.md]: same question - just the "update dependency" or the Dreamforce part as well?   If I'm removing the update dependency item, should I also get rid of the one on scripts for external dependency testing?

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 6 above)
Comment (vazexqi) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Just the LWC part. I think the UI in GitHub is not very precise when it comes to the line numbers.

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 5 above)
Comment (GaryFNC) [reply packages/salesforcedx-vscode/CHANGELOG.md]: could be either - but "debugger" is probably better

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 7 above)
Comment (vazexqi) [reply packages/salesforcedx-vscode/CHANGELOG.md]: The update dependency part and also get rid of the one on scripts for external dependency testing.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 8]
```diff
@@ -1,7 +1,16 @@
...
+## 41.16.0 - January 8, 2018
 ### Added
-#### salesforce-vscode-apex
-* Update apex-jorje-lsp.jar to enable go to definition for type usage sites ([PR #247] (https://github.com/forcedotcom/salesforcedx-vscode/pull/247))
+#### All
+* Upgrade to TypeScript 2.6.2 ([PR #242](https://github.com/forcedotcom/salesforcedx-vscode/pull/242))
+#### salesforcedx-vscode-apex
+* Enable Go To Definition for usages of classes and interfaces in class and interface declarations ([PR #247](https://github.com/forcedotcom/salesforcedx-vscode/pull/247))
+#### salesforcedx-apex-debugger
+* Add visual indication in the call stack telling the user what exception the debugger is currently paused on ([PR #240](https://github.com/forcedotcom/salesforcedx-vscode/pull/240)
```
Comment (ruthemmanuelle) [inline packages/salesforcedx-vscode/CHANGELOG.md]: There's a closing parenthesis missing for this item (there should be two parentheses after "240").

Comment (GaryFNC) [reply packages/salesforcedx-vscode/CHANGELOG.md]: thanks - oops....

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 9]
```diff
@@ -1,7 +1,16 @@
-##41.16.0 - January 8, 2018
+## 41.16.0 - January 8, 2018
 ### Added
-#### salesforce-vscode-apex
-* Update apex-jorje-lsp.jar to enable go to definition for type usage sites ([PR #247] (https://github.com/forcedotcom/salesforcedx-vscode/pull/247))
+#### All
```
Comment (vazexqi) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Remove lines 3 and 4. We don't need to announce that we upgrade TypeScript.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 10]
```diff
@@ -1,7 +1,16 @@
...
+* Upgrade to TypeScript 2.6.2 ([PR #242](https://github.com/forcedotcom/salesforcedx-vscode/pull/242))
+#### salesforcedx-vscode-apex
+* Enable Go To Definition for usages of classes and interfaces in class and interface declarations ([PR #247](https://github.com/forcedotcom/salesforcedx-vscode/pull/247))
+#### salesforcedx-apex-debugger
```
Comment (vazexqi) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Add a space after each item here. Take a look at the previous entries to see how spacing is done.

**Pull request merged**